src/dev/typescript/exec_in_projects.ts

```diff
@@ -47,7 +49,7 @@ export function execInProjects(
       title: project.name,
     })),
     {
-      concurrent: true,
+      concurrent: Math.min(4, (os.cpus() || []).length / 2 || 1) || false,
```

Comment:
I think in case `os.cpus` returns only an array with 1 core, we will be setting `concurrent` to `0.5`